components/camel-paho-mqtt5/src/main/java/org/apache/camel/component/paho/mqtt5/PahoMqtt5Configuration.java

```diff
@@ -88,6 +88,8 @@ public class PahoMqtt5Configuration implements Cloneable {
     private int executorServiceTimeout = 1;
     @UriParam(defaultValue = "-1")
     private long sessionExpiryInterval = -1;
+    @UriParam(defaultValue = "false")
+    private boolean manualAcksEnabled = false;
```

Comment:
a boolean is false by default, so remove

Reply:
Good point. Removed.